ts/components/services/ServicesHomeButton.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { constNull } from "fp-ts/lib/function";
+import { connect } from "react-redux";
+import FooterWithButtons from "../ui/FooterWithButtons";
+import I18n from "../../i18n";
+import { GlobalState } from "../../store/reducers/types";
+import { Dispatch } from "../../store/actions/types";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const ServicesHomeButton = (props: Props): React.ReactElement => (
```

Comment:
```suggestion
// this component shows two CTA to enable or disable services
const ServicesEnablingFooter = (props: Props): React.ReactElement => (
```
Could this name be more descriptive? Usually the _home button_ is a control used to reach the home of the app/section